pkg/discord/main.go

```diff
@@ -804,6 +842,21 @@ func contains(slice []string, value string) bool {
 	return false
 }
 
+func orEmpty(str, defaultStr string) string {
+	if str == "" {
+		return defaultStr
+	}
+	return str
+}
+
+func argCountFmt(argCount int) string {
```

Comment:
we can probably drop the else here
```suggestion
func argCountFmt(argCount int) string {
	if argCount == 1 {
		return "∞"
	}
	
	return strconv.Itoa(argCount)	
}
```

Reply:
Fixed!